README.md

```diff
@@ -40,6 +40,11 @@ var res = md.render(src);
 console.log(res);
 ```
 
+## Options
+
+Enabling the `dataType` boolean flag in the options will generate figure tags that
+also declare the data-type being wrapped, e.g.: `<figure data-type="image">`. 
+This can be useful for applying special styling.
```

Comment:
You have other types of figures that are not images? This can also be acheived with `![](img.png){data-type=image}` when using [markdown-it-attrs](https://github.com/arve0/markdown-it-attrs), but if it's a common use case it's easily added.